indico/modules/events/papers/controllers/management.py

```diff
@@ -35,9 +40,11 @@ class RHPapersDashboard(RHManagePapersBase):
 
     def _process(self):
         if not self.event_new.has_feature('papers'):
-            return WPManagePapers.render_template('management/disabled.html', self._conf, event=self.event_new)
+            return WPManagePapers.render_template('management/disabled.html', self._conf, event=self.event_new,
+                                                  cfp=self.event_new)
```

Comment:
Yeah actually in disabled.html we don't even need it, I thought I'm passing to overview. My bad.